sdk/communication/communication-chat/src/models/models.ts

```diff
@@ -1,86 +1,95 @@
+  sender?: CommunicationIdentifier;
+  /** The timestamp (if applicable) when the message was deleted. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. */
+  deletedOn?: Date;
+  /** The last timestamp (if applicable) when the message was edited. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. */
+  editedOn?: Date;
 }
 
-export interface ChatMessageContent extends Omit<RestChatMessageContent, "participants"> {
-  /**
-   * Chat message content for type "participantAdded" or "participantRemoved" messages.
-   */
+/** Content of a chat message. */
+export interface ChatMessageContent {
+  /** Chat message content for messages of types text or html. */
+  message?: string;
+  /** Chat message content for messages of type topicUpdated. */
+  topic?: string;
+  /** Chat message content for messages of types participantAdded or participantRemoved. */
   participants?: ChatParticipant[];
+  /** Identifies a participant in Azure Communication services. A participant is, for example, a phone number or an Azure communication user. This model must be interpreted as a union: Apart from rawId, at most one further property may be set. */
+  initiator?: CommunicationIdentifier;
 }
 
-/**
- * An interface representing a chat thread.
- */
-export interface ChatThread extends Omit<RestChatThread, "createdBy"> {
-  /**
-   * The CommunicationUserIdentifier that identifies this chat thread owner.
-   */
-  readonly createdBy?: CommunicationUserIdentifier;
+/** A chat message read receipt indicates the time a chat message was read by a recipient. */
+export interface ChatMessageReadReceipt {
+  /** Identifies a participant in Azure Communication services. A participant is, for example, a phone number or an Azure communication user. This model must be interpreted as a union: Apart from rawId, at most one further property may be set. */
+  sender: CommunicationIdentifier;
+  /** Id of the chat message that has been read. This id is generated by the server. */
+  chatMessageId: string;
+  /** The time at which the message was read. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. */
+  readOn: Date;
 }
 
-/**
- * A participant of the chat thread.
- */
-export interface ChatParticipant extends Omit<RestChatParticipant, "id"> {
-  /**
-   * The CommunicationUserIdentifier that identifies this chat participant.
-   */
-  user: CommunicationUserIdentifier;
+/** A participant of the chat thread. */
+export interface ChatParticipant {
+  /** Identifies a participant in Azure Communication services. A participant is, for example, a phone number or an Azure communication user. This model must be interpreted as a union: Apart from rawId, at most one further property may be set. */
+  id: CommunicationIdentifier;
```

Comment:
```suggestion
  id: CommunicationIdentifierKind;
```

Reply:
Why we are using `CommunicationIdentifierKind` here?